build.gradle

```diff
@@ -71,8 +71,6 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "org.jetbrains.kotlin:kotlin-serialization:$kotlin_version"
         classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:$bintray_version"
-
-        // Protobuf is udes in JVM tests
```

Comment:
Why remove the comment?

Reply:
Because it's obvious from the presence of `protobuf` module